src/test/java/com/monov/student/controller/StudentControllerIntTest.java

```diff
@@ -0,0 +1,100 @@
+package com.monov.student.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.monov.commons.dto.ItemIds;
+import com.monov.student.StudentServiceApplication;
+import com.monov.student.entity.Student;
+import com.monov.student.service.StudentService;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlConfig;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.Arrays;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
Please do a Google research on how to turn of wild card imports (star imports) in IntelliJ Idea. Once turned off, go over all you classes and optimize imports in order to remove them.

Reply:
done